tasks/ember_templates.js

```diff
@@ -37,7 +37,7 @@ function manualCompile(templateCompilerPath, template, grunt){
   vm.runInContext(templateCompilerJs, context, 'ember-template-compiler.js');
 
   // Compile the template
-  vm.runInContext('compiledJS = (module.exports || exports).precompile(template);', context);
+  vm.runInContext('compiledJS = ((typeof Ember !== "undefined" && Ember.HTMLBars) || module.exports || exports).precompile(template);', context);
```

Comment:
@dgeb Actually, the real change is only this few characters, however testing with different versions with Ember is brute force, the expected result files are massive, but at least the test emulate a realistic environment. 

I guess, there will be a big change to keep the compatibility with Ember v2.9. :)